### DISCUSSION TITLE
2023/10/artists/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#96] [General]
# 2023/10/artists/

https://xiangyun.rbind.io/2023/10/artists/

**Comment by earfanfan** [comment]
我最近读陶勇医生的《目光》，还有今天下午看的《珍·古道尔的传奇一生》，他/她的人生一点都不虚空或者虚无。虚无主义的提出，本意并不是让人陷入虚无主义的圈套，而是另一种。感到虚无可能是因为身不由己、己不由心，没有找到愿意付出全部的热情、耐心去做的事。不过好像很少人像他/她那么幸运，那么早就找到了，但也可以在探索的路上慢慢寻找。

另，AI 已经变革了我们的生活，它本来就是来源于人类自身的需求。
